doc/application/index.rst

```diff
@@ -534,7 +534,7 @@ at CMake configure time if any experimental feature is enabled.
    CONFIG_WARN_EXPERIMENTAL=y
 
 For example, enabling experimental warnings and building a sample which enables
-:kconfig:`CONFIG_BT_EXT_ADV` will print the following warning at CMake
+:kconfig:`CONFIG_BT_PER_ADV` will print the following warning at CMake
```

Comment:
A couple of questions about this one:

1. Why should this be squashed into `nrf noup] Bluetooth: update experimental for qualificatio`? According to `git blame`, the line was originally added in commit f17630ba750, which is upstream.
2. Why is CONFIG_BT_EXT_ADV wrong? I see that option used throughout the zephyr tree
3. Did you forget to update the code block on line 542 also?

Reply:
Thanks for the questions, Martin. I will do my best to answer them.

1. This is something I am not entirely sure I understand, nor can answer with confidence... I am open to suggestions though.
2. This was recommended by a PM in the Technical Support Team/tech-pubs Teams channel, on 17th of Dec. In the original suggestion, it was suggested ```CONFIG_BT_EXT_ADV``` maybe still be experimental, and the other config option was recommended.
This is something I did not double-check... My apologies. What are your thoughts about this @mbolivar-nordic? And please feel free to let me know if there are others I should include in this conversation. Thank you!

3. Whoops! Yes, thank you for checking that. I will edit that, if we all agree by then end that this should be changed.

Reply:
> This is something I am not entirely sure I understand, nor can answer with confidence... I am open to suggestions though.

@tejlmand can you respond here? This looks suspicious to me but I am not sure what the right way forward is here.

Reply:
> @tejlmand can you respond here? This looks suspicious to me but I am not sure what the right way forward is here.

@mbolivar-nordic I believe I might have been asking @kittydepa to use the `squash` info.
It is correct that upstream uses `CONFIG_BT_EXT_ADV` when showing how the `EXPERIMENTAL` feature work, if you read the whole part:
> For example, enabling experimental warnings and building a sample which enables
> :kconfig:`CONFIG_BT_EXT_ADV` will print the following warning at CMake
> configure time.
>
> .. code-block:: shell
> warning: Experimental symbol BT_EXT_ADV is enabled.

However, trying this with NCS will not print that warning, because in this `[noup]` commit, we remove `select EXPERIMENTAL` on `BT_EXT_ADV`: https://github.com/nrfconnect/sdk-zephyr/commit/ec6d4e8185295f48e9255128966469240932c4b7

Therefore we must update the docs to show case another `EXPERIMENTAL` symbol.
And therefore I think this commit ought to be squashed into the commit that is the root cause for this symbol no longer being experimental.